src/geovista/bridge.py

```diff
@@ -636,6 +652,42 @@ def from_points(
             mesh.field_data[GV_FIELD_NAME] = np.array([name])
             mesh[name] = data
 
+        if vectors is not None:
+            if vectors_array_name is None:
+                vectors_array_name = "vectors"
+
+            if not isinstance(vectors, tuple) or not all(
+                isinstance(arr, np.ndarray) for arr in vectors
+            ):
+                msg = 'Keyword "vectors" must be a tuple of array-like.'
+                raise ValueError(msg)
+
+            n_vecs = len(vectors)
+            if n_vecs not in (2, 3):
+                msg = (
+                    'Keyword "vectors" must be a tuple of 2 or 3 arrays : '
+                    f"got {n_vecs}."
+                )
+                raise ValueError(msg)
+
+            vectors = [np.asanyarray(vecdata) for vecdata in vectors]
+            xx, yy = vectors[:2]
+            zz = vectors[2] if n_vecs > 2 else np.zeros_like(xx)
+
+            if vectors_z_scaling is not None:
+                zz = zz * vectors_z_scaling
```

Comment:
Interesting...
I'm actually doubting right now whether we may not remove this feature **_altogether_**, since it's so obvious how the user would do it.
I already removed an overall scaling operation for the same reason, and also because it can be done in the 'glyph' call.